gg-pac-team3-front/src/pages/frontPage/FrontPage.jsx

```diff
@@ -0,0 +1,12 @@
+import NavBlue from "../../components/layout/navBlue";
```

Comment:
Similarly, `navBlue` here should be `NavBlue`, the name of our component, for this import to work.